examples/js/controls/TransformControls.js

```diff
@@ -161,36 +165,45 @@
 					child.position.set( 0, 0, 0 );
 					child.rotation.set( 0, 0, 0 );
 					child.scale.set( 1, 1, 1 );
+
 				}
+
 			});
 
 		};
 
 		this.hide = function () {
-			this.traverse(function( child ) {
```

Comment:
Since visible is a public property makes me wonder why we'd want `show` and `hide` methods on the internal classes at all.